### PR TITLE
Fv southern tutchone

### PR DESCRIPTION
**Pull request by HopsAndHops**
- Updated to single codepoint outputs where possible
- Updated documentation
- Centered keyboard
- Removed duplicate punctuation

Approved by @caforbes

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/fv/fv_southern_tutchone/source/fv_southern_tutchone.kmn [hunk 1]
```diff
@@ -8,24 +8,150 @@ store(&HOTKEY) '[CTRL SHIFT K_1]'
...
+'ą̈̀' + any(MBackspace) > 'ą̈'
+'ų̈́' + any(MBackspace) > 'ų̈'
+'ų̈̀' + any(MBackspace) > 'ų̈'
+'Ą̈́' + any(MBackspace) > 'Ą̈'
+'Ą̈̀' + any(MBackspace) > 'Ą̈'
+'Ų̈́' + any(MBackspace) > 'Ų̈'
+'Ų̈̀' + any(MBackspace) > 'Ų̈'
+
+
 store(&LAYOUTFILE) 'fv_southern_tutchone.keyboard_info'
```
Comment (darcywong00) [inline release/fv/fv_southern_tutchone/source/fv_southern_tutchone.kmn:152]: > fixup &layoutfile reference in fv_southern_tutchone.kmn

Regarding my other comment on the pull request about renaming files

```suggestion
store(&LAYOUTFILE) 'fv_southern_tutchone.keyman-touch-layout'
```

**Comment by darcywong00** [comment]
Thank you for the keyboard submission!

There's some cleanup of filenames as follows:

> fv_southern_tutchone:
a. delete source/fv_southern_tutchone.keyman-touch-layout
b. rename source/fv_southern_tutchone.keyboard_info to source/fv_southern_tutchone.keyman-touch-layout
c. fixup &layoutfile reference in fv_southern_tutchone.kmn
d. ask FV team to verify the touch layout is correct.

**Comment by HopsAndHops** [comment]
Thanks, Darcy! Made the requested changes. The keyboard was approved by @caforbes

**Review by LornaSIL — APPROVED** [review]
LGTM.

**Pull request merged**